lib/Log/Any/Adapter/DERIV.pm

```diff
@@ -229,8 +246,25 @@ sub apply_filehandle_utf8 {
     $fh->autoflush(1);
 }
 
+=head2 format_line
+
+Format the log entry
+
+=over 4
+
+=item * C<$data> log data -severity, message, stack
+
+=item * C<$opts> options - color
+
+=back
+
+Returns the color details for log
```

Comment:
what are color details?

I'd suggest:

```
Returns Unicode string with embedded ANSI colour codes.
```

(or "UTF-8 byte string", if that's what it returns)